src/js/va-letters/reducers/index.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+
+import { UPDATE_ADDRESS_CONFIRMATION } from '../actions/index';
+
+const initialState = {
+  addressConfirmed: null
+};
+
+export default function userReducer(state = initialState, action) {
+  let newState = undefined;
+
+  switch (action.type) {
+    case UPDATE_ADDRESS_CONFIRMATION: {
+      newState = Object.assign({}, state);
+      _.set(newState, 'addressConfirmed', action.value);
```

Comment:
Since`lodash/fp` copies the state and returns a new one, I think you can just return this and get rid of `newState`